roxie/ccd/ccdprotocol.cpp

```diff
@@ -1681,27 +1681,19 @@ class RoxieSocketWorker : public ProtocolQueryWorker
         return id;
     }
 
-    const char *queryRequestGlobalIdHeader(HttpHelper &httpHelper, IContextLogger &logctx, StringAttr &headerused)
+    const char *queryRequestGlobalIdHeader(HttpHelper &httpHelper, IContextLogger &logctx)
     {
-        const char *id = queryRequestIdHeader(httpHelper, logctx.queryGlobalIdHttpHeaderName(), headerused);
+        const char *id = httpHelper.queryRequestHeader("Global-Id");
         if (!id || !*id)
-        {
-            id = queryRequestIdHeader(httpHelper, "Global-Id", headerused);
-            if (!id || !*id)
-                id = queryRequestIdHeader(httpHelper, "HPCC-Global-Id", headerused);
-        }
+            id = httpHelper.queryRequestHeader("HPCC-Global-Id");        // Backward compatibility - passed on as global-id
         return id;
     }
 
-    const char *queryRequestCallerIdHeader(HttpHelper &httpHelper, IContextLogger &logctx, StringAttr &headerused)
+    const char *queryRequestCallerIdHeader(HttpHelper &httpHelper, IContextLogger &logctxheaderused)
     {
-        const char *id = queryRequestIdHeader(httpHelper, logctx.queryCallerIdHttpHeaderName(), headerused);
+        const char *id = httpHelper.queryRequestHeader("Caller-Id");
```

Comment:
I'm sure there's a good reason we're not using the definition from jtrace.hpp, but pointing it out in case we want/can avoid all these literals.

Reply:
as above - the reason was those definitions didn't exist.